doc/nrf/gs_testing.rst

```diff
@@ -120,3 +120,74 @@ To connect to the nRF9160-based kit with LTE Link Monitor, perform the following
    .. note::
 
       In the case of nRF9160 DK, the reset button must be pressed to restart the device and to start the application.
+
+.. _gs_debugging:
+
+Debugging an application
+************************
+
+To debug an application, set up the debug session as described in `Debugging an application`_ in the |nRFVSC| documentation.
+
+If you use a multi-core SoC, for example from the nRF53 Series, you need a single debug session to debug the application core firmware.
+To debug the firmware running on the network core, you need to set up two separate debug sessions: one for the network core and one for the application core.
+When debugging the network core, the application core debug session runs in the background and you can debug both cores if needed.
+
+Complete the following steps to start debugging the network core:
+
+1. Set up sessions for the application core and network core as mentioned in `Debugging an application`_.
+#. Select the appropriate CPU for debugging in each session, corresponding to the application core and the network core of your SoC, respectively.
+#. Once both sessions are established, execute the code on the application core.
+
+   The startup code releases the ``NETWORK.FORCEOFF`` signal to start the network core and allocates the necessary GPIO pins for it.
+#. Start code execution on the network core in the other debug session.
+
+If you want to reset the network core while debugging, make sure to first reset the application core and execute the code.
+
+.. note::
+  Debugging firmware on the application core in the non-secure domain (that is, when using the ``_ns`` build target), is currently not supported.
+
+Debug configuration
+===================
+
+The following Kconfig options are used by the |VSC| debugger in the |nRFVSC|:
+
+* :kconfig:option:`CONFIG_DEBUG_OPTIMIZATIONS` set to ``y``
+* :kconfig:option:`CONFIG_DEBUG_THREAD_INFO` set to ``y``
+
+Debug logging
+-------------
+
+Some protocols, applications, and samples may require additional Kconfig options to be set when debugging.
+Among these are logging backend configuration options that need to be set for the logging system to provide debug logs.
+For example, nRF9160's :ref:`nRF Cloud MQTT multi-service <nrf_cloud_mqtt_multi_service>` sample will start printing debug logs after you enable :ref:`CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_DBG <CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_DBG>`.
```

Comment:
```suggestion
For example, the :ref:`nRF Cloud MQTT multi-service <nrf_cloud_mqtt_multi_service>` nRF9160 sample will start printing debug logs after you enable :ref:`CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_DBG <CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_DBG>`.
```
This could also work with just "the :ref:\`nrf_cloud_mqtt_multi_service\` sample", I think.

Reply:
Removed this line.